src/sass/_normalize.scss

```diff
@@ -0,0 +1,12 @@
+// Ideally all of this will go in uswds/src/stylesheets/lib/normalize, but until
+//  it does, normalizing css goes in here.
+
+dl {
```

Comment:
The only other place I saw a `<dl>` was in the [Rx GlossaryList](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/components/GlossaryList.jsx#L25). It should be fine since it's just normalizing. Takes what Chrome was already doing and making sure other browsers do the same.